fastseq_cli/transformers_generate.py

```diff
@@ -207,30 +212,32 @@ def generate_summaries_or_translations(
         tokenizer = AutoTokenizer.from_pretrained(model_name)
 
     if fp16:
-        model = model.half()
+        model = model.half()  
     if decoder_start_token_id is None:
         decoder_start_token_id = gen_kwargs.pop("decoder_start_token_id", None)
-
     if hasattr(tokenizer, 'model_max_length') and max_tokenizer_length is not None:
         tokenizer.model_max_length = max_tokenizer_length
 
     # update config with summarization specific params
     use_task_specific_params(model, task)
-    data_queue = Queue()
-    msg_queue =  Queue()
-    p_list = []
 
-    for _ in range(postprocess_workers):
-        p = PostProcess(tokenizer, data_queue, msg_queue,
-            skip_special_tokens, clean_up_tokenization_spaces)
-        p_list.append(p)
-        p.start()
+    if fastseq_opt:
+        data_queue = Queue()
+        msg_queue =  Queue()
+        p_list = []
+
+        for _ in range(postprocess_workers):
+            p = PostProcess(tokenizer, data_queue, msg_queue,
+                skip_special_tokens, clean_up_tokenization_spaces)
+            p_list.append(p)
+            p.start()
+
+        io_process = IOProcess( msg_queue, fout)
+        io_process.start()
 
-    io_process = IOProcess( msg_queue, fout)
-    io_process.start()
     dataset = TokenizeDataset(examples, tokenizer, model_name,
-        model.config.prefix, return_tensors, truncation, padding,
-        max_tokenizer_length)
+            model.config.prefix, return_tensors, truncation, padding,
+            max_tokenizer_length)
```

Comment:
nit: Are these format changes needed?

Reply:
not sure why they're there. I don't see them on my end. Something must be off w/ my tab/space settings.